### DISCUSSION TITLE
Can't use CrossBleAdapter

### DISCUSSION THREAD

**Discussion by AlessandroFinocchi** [shinyorg/shiny#1057] [Q&A]
Hi everyone, I'm approching MAUI Blazor for a project which needs the implementation of a bluetooth scanner and for this I was following the Shiny guide "https://shinylib.net/mobile/ble/scanning.html". So I went in the section "Bluetooth LE Client" but there I found the usage of the CrossBleAdapter which I can't use because it is defined nowhere.
This is just a case of something used in the guide but not present in any of the Nuget package that the guide wants you to install.
Did I miss anything?
Any kind of advice will be really appreciated. Thx <3.

**Comment by aritchie** [comment]
The documentation is far from finished.  The bluetooth docs are from the old plugin.  There are samples here: https://github.com/shinyorg/samples
